src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1189,7 +1201,7 @@ class AbstractUtxoCoin extends BaseCoin {
    * - krsProvider: necessary if backup key is held by KRS
    * - recoveryDestination: target address to send recovered funds to
    * - scan: the amount of consecutive addresses without unspents to scan through before stopping
-   * - ignoreAddressTypes: (optional) array of AddressTypes to ignore. these are strings defined in Codes.UnspentTypeTcomb
+   * - ignoreAddressTypes: (optional) array of AddressTypes to ignore, these are strings defined in Codes.UnspentTypeTcomb
```

Comment:
we should set a default value here - if ignoreAddressTypes is not passed in, it should by default be set to an array of address types that we should ignore for that specific coin.
so we may need to add a field on each coin that is something like "coin.ignoreRecoveryAddressTypes" or "coin.addressTypesUsed" or something like that

Reply:
the reason is because i just ran a test recovery with Zcash, and the public block explorers were throwing errors because we were sending them requests for bech32 zcash addresses.

Reply:
Did not use `ignoreAddressTypes` but my new commit should fix this issue.

Reply:
nice looks good